### ISSUE TITLE
`import imp` is deprecated

### ISSUE THREAD

**Issue by reynoldsnlp** [swig/swig#1621]
I'm using the `hfst` python package, which is compiled using `swig`, and I get the following `DeprecationWarning`:

```
DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
```

Looking at the SWIG-generated file...

```python
from sys import version_info
if version_info >= (2, 6, 0):
    def swig_import_helper():
        from os.path import dirname
        import imp
```

**Comment by reynoldsnlp** [comment]
It occurs to me that this might be an issue that @hfst hasn't recompiled with a newer version of SWIG. If that's the case, then I will poke them.

**Comment by ojwb** [comment]
Indeed - this was fixed in SWIG 4.0.0.
